File: RelativeRanks/RelativeRanks.py
class Solution:
    def findRelativeRanks(self, score: List[int]) -> List[str]:
        arr = sorted(score, reverse=True)
        l = len(arr)
        dict_ = {}
        dict_[arr[0]] = "Gold Medal"
        if l > 1:
            dict_[arr[1]] = "Silver Medal"
        if l > 2:
            dict_[arr[2]] = "Bronze Medal"
        for i in range(3, len(arr)):
            dict_[arr[i]] = str(i + 1)
        answer = [dict_[k] for k in score]
        return answer
            
            

File: RelativeRanks/RelativeRanks.js
/**
 * @param {number[]} score
 * @return {string[]}
 */
var findRelativeRanks = function (score) {
  let ranks = score.slice(0).sort((a, b) => b - a);
  return score.map((num, i) => {
    if (num === ranks[0]) return "Gold Medal";
    else if (num === ranks[1]) return "Silver Medal";
    else if (num === ranks[2]) return "Bronze Medal";
    else return (ranks.indexOf(num) + 1).toString();
  });
};
